spec/standard-library.md

```diff
@@ -1,730 +1,535 @@
-# Standard library {#standard-library .Appendix1}
+# Standard library
+
+## General
+
+A conforming C# implementation shall provide a minimum set of types having specific semantics. These types and their members are listed here, in alphabetical order by namespace and type. For a formal definition of these types and their members, refer to ISO/IEC 23271:2012 *Common Language Infrastructure (CLI), Partition IV; Base Class Library (BCL), Extended Numerics Library, and Extended Array Library*, which are included by reference in this specification.
+
+> [!NOTE]
```

Comment:
@v-kents was there a Note around the informative text? Please see [https://review.docs.microsoft.com/en-us/content-production-service/internal/c-sharp-language-guide?branch=c-sharp-task#normative-and-informative-text](https://review.docs.microsoft.com/en-us/content-production-service/internal/c-sharp-language-guide?branch=c-sharp-task#normative-and-informative-text). Please also see lindexi comment: [https://github.com/dotnet/csharplang/pull/3136/files#r373770237](https://github.com/dotnet/csharplang/pull/3136/files#r373770237). Once these are resolved, then I believe that this PR is ready for Bill Wagner to review. Nice work!

Reply:
I was going off of Any instance of the above should at minimum be indicated with a blockquote (> on each line of the informative text), with the, so I guess it needs the >, but no NOTE header?